src/site/content/en/fast/minify-css/index.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Minify CSS
+author: demianrenzulli
+description: |
+  Learn how to minify CSS files, to improve performance, without affecting how the browser process the styles.
+web_lighthouse:
+- render-blocking-resources
+date: 2019-05-02
+---
+
+CSS files can contain unnecessary characters, such as comments, whitespaces, and indentation. 
+In production, these characters can be safely removed, to reduce file size without affecting how the browser processes the styles. This technique is called **minification**.
+
+## Loading unminified CSS
+
+Take a look at the following CSS block:
+
+```
+body {
+  font-family: "Benton Sans", "Helvetica Neue", helvetica, arial, sans-serif;
+  margin: 2em;
+}
+
+/* all titles need to have the same font, color and background */
+h1 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+
+h2 {
+  font-style: italic;
+  color: #373fff;
+  background-color: #000000;
+}
+```
+
+This content is easy to read, at the cost of producing a larger than necessary file:
+
+- It uses spaces for indentation purposes and contains comments, which are ignored by the browser, so they can be removed.
+- The `<h1>` and `<h2>` elements have the same styles: instead of declaring them separately: "`h1 {...} h2 {...}`" they could be expressed as "`h1, h2{...}`".
+- The **background-color**, `#000000` could be expressed as just `#000`.
+
+After making these changes, you would obtain a more compact version of the same styles:
+
+```
+body{font-family:"Benton Sans","Helvetica Neue",helvetica,arial,sans-serif;margin:2em}h1,h2{font-style:italic;color:#373fff;background-color:#000}
+```
+
+You probably don't want to write CSS like that. Instead, you can write CSS as usual, and add a minification step to your build process. In this guide, you’ll learn how to do it by using a popular build tool: [Webpack](https://webpack.js.org/).
+
+## Measure
+
+You’ll apply CSS minification to a site that has been used in other guides: [Fav Kitties](https://fav-kitties-animated.glitch.me/). This version of the site uses a cool CSS library: [animate.css](https://github.com/daneden/animate.css), to animate different page elements when a user votes for a cat 😺.
+
+As a first step, you need to understand what would be the opportunity after minifying this file:
+
+1. Open [Lighthouse](https://web.dev/measure). 
+1. Enter the URL: https://fav-kitties-animated.glitch.me/ and click **Run Audit**. 
+1. Click **View report**.
+1. Click on **Performance** and go the **Opportunities** section.
+
+The resulting report shows that up to **16KB** can be saved from the **animate.css** file:
+
+<img class="screenshot" width="700px" height="150px" src="./lighthouse-unoptimized.png" alt="Lighthouse - Minify CSS opportunity.">
+
+Now you'll inspect the content of the CSS:
+
+1. Open the [site](https://fav-kitties-animated.glitch.me/) in Chrome (it might take a while for Glitch servers to respond the first time).
```

Comment:
```suggestion
1. Open [the Fav Kitties site](https://fav-kitties-animated.glitch.me/) in Chrome (it might take a while for Glitch servers to respond the first time).
```